core/src/main/java/discord4j/core/object/entity/TextChannel.java

```diff
@@ -256,88 +67,8 @@ public int getRateLimitPerUser() {
-                                .subscriberContext(ctx -> ctx.put("shard",
-                                        getServiceMediator().getClientConfig().getShardIndex())))
-                        .hasElement()
-                        .map(identity -> !identity);
-
-        return Flux.defer(() -> messageIds)
-                .distinct()
-                .filter(filterMessageId)
-                .map(Snowflake::asString)
-                .buffer(100) // REST accepts 100 IDs
-                .filterWhen(filterMessageIdChunk)
-                .map(messageIdChunk -> messageIdChunk.toArray(new String[messageIdChunk.size()]))
-                .flatMap(messageIdChunk -> getServiceMediator().getRestClient().getChannelService()
-                        .bulkDeleteMessages(getId().asLong(), new BulkDeleteRequest(messageIdChunk))
-                        .subscriberContext(ctx -> ctx.put("shard",
-                                getServiceMediator().getClientConfig().getShardIndex())))
-                .thenMany(Flux.fromIterable(ignoredMessageIds));
-    }
-
-    /**
-     * Requests to create a webhook.
-     *
-     * @param spec A {@link Consumer} that provides a "blank" {@link WebhookCreateSpec} to be operated on.
-     * @return A {@link Mono} where, upon successful completion, emits the created {@link Webhook}. If an error
-     * is received, it is emitted through the {@code Mono}.
-     */
-    public Mono<Webhook> createWebhook(final Consumer<? super WebhookCreateSpec> spec) {
-        final WebhookCreateSpec mutatedSpec = new WebhookCreateSpec();
-        spec.accept(mutatedSpec);
-
-        return getServiceMediator().getRestClient().getWebhookService()
-                .createWebhook(getId().asLong(), mutatedSpec.asRequest(), mutatedSpec.getReason())
-                .map(WebhookBean::new)
-                .map(bean -> new Webhook(getServiceMediator(), bean))
-                .subscriberContext(ctx -> ctx.put("shard", getServiceMediator().getClientConfig().getShardIndex()));
-    }
-
-    /**
-     * Requests to retrieve the webhooks of the channel.
-     *
-     * @return A {@link Flux} that continually emits the {@link Webhook webhooks} of the channel. If an error is
-     * received, it is emitted through the {@code Flux}.
-     */
-    public Flux<Webhook> getWebhooks() {
-        return getServiceMediator().getRestClient().getWebhookService()
-                .getChannelWebhooks(getId().asLong())
-                .map(WebhookBean::new)
-                .map(bean -> new Webhook(getServiceMediator(), bean))
-                .subscriberContext(ctx -> ctx.put("shard", getServiceMediator().getClientConfig().getShardIndex()));
-    }
-
     @Override
     public String toString() {
-        return "TextChannel{" +
-                "guildChannel=" + guildChannel +
-                ", messageChannel=" + messageChannel +
-                "} " + super.toString();
+        return "TextChannel{} " + super.toString();
```

Comment:
Can we change this format? It seems weird to me.

Reply:
We'll do this separately.

Reply:
Agree since we have this in many places as it's what intellij generates for subclasses